src/core/client_config/lb_policies/protos/load_balancer.proto

```diff
@@ -0,0 +1,115 @@
+syntax = "proto3";
```

Comment:
@a11r, @ejona86: where's the current best-practice place to locate .proto files?